web/admin/src/ContentDetailsEditor.js

```diff
@@ -53,7 +53,16 @@ export default class ContentDetailsEditor extends PureComponent {
         </div>
     </div>
       case 'survey': return <div>
-        TODO: Survey UI
+        <div>
+            <h2>Add a Title</h2>
+            <TextEditor content={c} prop="title" title="Title" placeholder="Acme Co Name" onUpdate={onUpdate} />
+          </div>
+          <div className="homeBox">
+            <h2>Add Content</h2>
+            <TextEditor content={c} prop="text" title="Content" placeholder="Acme Co Details" onUpdate={onUpdate} />
+            <TextEditor content={c} prop="url" title="URL" placeholder="http://www.acme.com" onUpdate={onUpdate} />
```

Comment:
These looks like web view fields, not survey fields.

Reply:
I'm not familiar with our surveys but do they have titles and content already to match the mock ups?

Reply:
Correct, all that we will need here is a selector to pick from the global surveys from the CMS